composer.json

```diff
@@ -18,7 +18,8 @@
     "fakerphp/faker": "^1.15",
     "guzzlehttp/psr7": "^1.7",
     "phpstan/phpstan": "^0.12.89",
-    "squizlabs/php_codesniffer": "^3.6"
+    "squizlabs/php_codesniffer": "^3.6",
+    "nikic/php-parser": "^4.12"
```

Comment:
Needs to be added for `prefer-lowest` installations, since there was a php 8.1 related bugfix that otherwise caused this output when running phpunit with code coverage:
```
Runtime:       PHP 8.1.0RC2-dev with Xdebug 3.2.0-dev
Configuration: /home/runner/work/json-api-common/json-api-common/phpunit.xml.dist

Undefined array key 288
Error: Process completed with exit code 2.
```
I wasn't able to discover the exact reason though. Only that its gone with `nikic/php-parser:4.12.0`.